server/lib/pbench/api/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
```

Comment:
why `server/lib/pbench/api` ?

I suppose it's not a conflict on the 0.69 branch; but it's going to be confusing moving back and forth since it means the gunicorn web server "API" on main.

Reply:
`conftest.py` needed `get_server_config` and it was in the `API` folder so I let it be in it.

Reply:
it is only used by `conftest.py` so moving it to `test` also makes sense.